data/dashboard.yml

```diff
@@ -8,6 +8,7 @@
     - name: Frontend apps
     - name: data.gov.uk apps
     - name: Licensing apps
+    - name: Specialist apps
```

Comment:
Perhaps 'Service apps' would be more specific, since this is how we refer to apps by other departments that run on `.service.gov.uk`

Reply:
There's already a "Services" category - it might be a bit confusing to have "Service apps" and "Services".

I thought about "Coronavirus apps", but maybe that's too on-the-nose?